guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc

```diff
@@ -77,7 +77,7 @@ Therefore, do not delete, block, or change permissions of the user during the to
 If you keep this field blank, {Project} uses the default network interface.
 
 ifeval::["{build}" == "satellite"]
-. In the *Activation Key(s)* field, enter one or more activation keys to assign to hosts.
+. Optional: In the *Activation Key(s)* field, enter one or more activation keys to assign to hosts if you are not using a Host Group with an activation key.
```

Comment:
Activation key is always required, without it registration will fail:
```
Complete!
Error: Must specify an activation key
```

Getting AKs from host group will be supported in 6.10

Reply:
OK, I have removed that change, thank you